doc/api.md

```diff
@@ -387,6 +387,20 @@ in
 The exe produced by haskell.nix that you want to wrap in a container.
 
 
+---
+
+### `mkContainerFromCabalExe.<in>.name`
+
+**Type**: null or string
+
+**Default**: `exe.exeName`
```

Comment:
Note: My intentions are to not have `null` in the type. Only to have it as default to `exe.exeName` and settable any string.

Reply:
See https://github.com/input-output-hk/iogx/pull/26#discussion_r1422311644